api/app/Http/Controllers/Api/AuthController.php

```diff
@@ -77,4 +81,75 @@ public function register(UserRegistrationModel $request)
 
         User::create($account);
     }
+
+    /**
+     * Send a mail for resetting the password.
+     *
+     * @param \App\Http\Requests\Api\PasswordResetModel $request
+     *
+     * @return mixed
+     */
+    public function sendResetMail(PasswordResetModel $request)
+    {
+        $user_email = $request->email;
+        $user = User::where('email', $user_email)->first();
+        if ($user && $this->notSpamming($user, env('PASSWORD_RESET_MINUTES', 30))) {
+            // User exists and had no request < PASSWORD_RESET_MINUTES
+            $uuid_token = Uuid::generate(4);
```

Comment:
Please use camelCase :)
Also, because it's unique, it can still be bruteforced with ease.

str_random and an uuid combined would be more secure :)